src/command_modules/azure-cli-sql/azure/cli/command_modules/sql/tests/test_sql_commands.py

```diff
@@ -362,9 +382,8 @@ def body(self):
         self.cmd('sql elastic-pools db show-activity -g {} --server-name {} --elastic-pool-name {}'
                  .format(rg, self.sql_server_name, self.pool_name),
                  checks=[
-                     JMESPathCheck('[0].resourceGroup', rg),
-                     JMESPathCheck('[0].serverName', self.sql_server_name),
-                     JMESPathCheck('[0].currentElasticPoolName', self.pool_name)])
+                     JMESPathCheck('type(@)', 'array'),
+                     CustomCheck(self.custom_activity_checker)])
```

Comment:
@derekbekoe Can you please elaborate on the question  "work with other JMESPath checks?" means? The way I understand this is one more custom way to check for `checks` array list.